api/drivers/aws/driver.js

```diff
@@ -312,6 +312,67 @@ class AWSDriver extends Driver {
   }
 
   /**
+   * Start the specified machine.
+   *
+   * @method startMachine
+   * @return {Promise[Object]}
+   */
+  startMachine(machine) {
+    return new Promise((resolve, reject) => {
+      var params = {
+        InstanceIds: [
+          machine.id
+        ],
+        DryRun: false
+      };
+      this._client.ec2.startInstances(params, (err) => {
+        if (err) {
+          return reject(err);
+        } else {
+          params = {
+            DryRun: false,
+            InstanceIds: [
+              machine.id
+            ]
+          };
+          this._client.ec2.describeInstances(params, (err, data) => {
+            if (err) {
+              return reject(err);
+            } else {
+              machine.ip = data.Reservations[0].Instances[0].PrivateIpAddress;
+              return resolve(machine);
+            }
+          });
+        }
+      });
+    });
+  }
+
+  /**
+   * Stop the specified machine.
+   *
+   * @method stopMachine
+   * @return {Promise}
```

Comment:
This makes to API inconsistent. We should resolve with the machine even though it's shutdown.